lms/static/my.js

```diff
@@ -106,14 +106,39 @@ function trackReadAllNotificationsButton(button) {
   });
 }
 
+function getPostUploadMessage() {
+  const myDropzone = Dropzone.forElement('#demo-upload');
+  const feedbacks = document.getElementById('upload-feedbacks');
+  const matchesSpan = document.getElementById('upload-matches');
+  const missesSpan = document.getElementById('upload-misses');
+  myDropzone.on('success', function() {
+    const args = Array.from(arguments).slice(1)[0];
+    const matches = args['exercise_matches'];
+    const misses = args['exercise_misses'];
+    if (matches.length) {
+      matchesSpan.innerHTML += matches + ',';
```

Comment:
https://stackoverflow.com/questions/19030742/difference-between-innertext-innerhtml-and-childnodes-value

https://en.wikipedia.org/wiki/Cross-site_scripting